docs/app/pages/landing/landing.component.html

```diff
@@ -2,6 +2,13 @@
 
 <uxd-full-page-layout>
 
+    <ux-alert type="error">
+        <p>AngularJS 1.x has reached the end of its life as of the 31st of December 2021. For more information check the <a routerLink="/changelog">changelog</a></p>
+        <p>As a result no further updates will be made to this library. For the Angular version, see <a href="https://uxaspects.github.io/UXAspects/">uxaspects.github.io/UXAspects</a></p>
+    </ux-alert>
+
+    <br />
```

Comment:
Per previous advice, don't use `br` to add vertical space. Use the stylesheet or Bootstrap's utility classes to space elements.